exporter/awsprometheusremotewriteexporter/auth.go

```diff
@@ -100,6 +106,17 @@ func getCredsFromConfig(auth AuthConfig) *credentials.Credentials {
 	return creds
 }
 
+var regionRe = regexp.MustCompile(`aps-workspaces\.(.*)\.amazonaws\.com`)
+
+func parseEndpointRegion(endpoint string) (region string, err error) {
+	// Example: https://aps-workspaces.us-east-1.amazonaws.com/workspaces/ws-XXX/api/v1/remote_write
+	results := regionRe.FindAllStringSubmatch(endpoint, 1)
```

Comment:
Can we just use `FindString`?

Reply:
Removed the regexp, it is unnecessary. PTAL.